linkupdate.raml

```diff
@@ -0,0 +1,85 @@
+#%RAML 0.8
+title: Linkupdate
+version: v0.1.1-20180606
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2016, 2017, 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+
+schemas:
+  - linkupdate: !include schemas/oic.if-linkupdate-schema.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.linkupdate"]
+
+
+/LinkupdateResURI?if=oic.if.linkupdate:
+    description: |
+      To add links, send a link list format containing the links to be added.
+      To remove links, use query parameters to select link(s) to delete and send an empty link array [].
+      To modify links, use query parameters to select link(s) to patch and send the cbor.
+      Serialization of json merge-patch format using the linkupdate schema.
+    displayName: linkupdate
+
+    post:
+        description: |
+          Add, delete, or modify links using the UPDATE operation on the array of links
+
+        queryParameters:
+          href:
+            type: string
+          rel:
+            type: string
+          rt:
+            type: string
+          ins:
+            type: string
+          location-tag:
+            type: string
+
+        body:
+          application/json:
+            schema: linklist
+              example: |
+                [
+                  {
+                    "href": "ocf://5c68f373-0c4c-4604-9922-db639e20533d/switch",
+                    "rel": ["boundto"],
+                    "bind": "obs",
+                    "anchor": "modeswitch",
+                    "if": ["oic.if.s"],
+                    "ins": "377fac40"
+                  }
+                ]
+
+          application/merge-patch+json:
```

Comment:
This isn't the patch defined by CoAP, is it?  I'm not sure what "merge-patch+json" means.